var/spack/repos/builtin/packages/py-torch-nvidia-apex/package.py

```diff
@@ -14,11 +14,13 @@ class PyTorchNvidiaApex(PythonPackage, CudaPackage):
     phases = ['install']
 
     version('master', branch='master')
+    version('2020-10-19', commit='8a1ed9e8d35dfad26fb973996319965e4224dcdd')
 
     depends_on('python@3:', type=('build', 'run'))
     depends_on('py-setuptools', type='build')
     depends_on('py-torch@0.4:', type=('build', 'run'))
     depends_on('cuda@9:', when='+cuda')
+    depends_on('py-pybind11', when='@2020-10-19:', type=('build', 'link', 'run'))
```

Comment:
This is needed for all versions, right? I don't think `when=` is necessary then.

Reply:
If they need an older version, they can always add it to the package and add a `when` condition. Up to you I guess.

Reply:
eh... I removed it...